app/services/movement_service.rb

```diff
@@ -1,7 +1,7 @@
 class MovementService
   # rubocop:disable Metrics/MethodLength
   def self.movements_on(date, direction_filters: [], type_filters: [])
-    movements = Nomis::Elite2::Api.movements_on_date(date)
+    movements = Nomis::Api::MovementApi.movements_on_date(date)
```

Comment:
I think the split makes sense, but rather than have `Nomis::Api::MovementApi` I think it's worth being more explicit about the top level API that is being used i.e `Nomis::Elite2::MovementApi` so we know where we are accessing the API from.